app/controlplane/pkg/ca/ca.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ca
```

Comment:
I find `ca` to short and can be confusing with some other terms like `cas`. Can you think of a different `name`? If not that's ok.

Reply:
Yes, I'll figure out something

Reply:
You could call it signing_ca, issuing_ca, or something like that.

Reply:
Thanks, I'll create a follow-up task to improve the naming. `signing_ca` sounds good to me.